### DISCUSSION TITLE
Downgrading a Language Server version

### DISCUSSION THREAD

**Discussion by ggranberry** [mason-org/mason.nvim#1413] [Q&A]
Hey, so I'm troubleshooting some issues that I'm currently running into with JDTLS, and I figured I'd try downgrading to v1.24.0 since its a relatively new release. So I looked through the docs and found the `MasonInstall jdtls@v1.24.0` command. However it fails due to an incorrect hash (was expecting jdt-language-server-1.24.0-202306011728.tar.gz) but got 

```
 Fetching available versions…
        Downloading file "https://download.eclipse.org/jdtls/milestones/1.24.0/jdt-language-server-1.24.0-202306291518.tar.gz"…
        spawn: wget failed with exit code 8 and signal 0. 
        Failed to download file "https://download.eclipse.org/jdtls/milestones/1.24.0/jdt-language-server-1.24.0-202306291518.tar.gz".
```

Is there some way to include the file name or hash with this command?

**Comment by zenoli** [comment]
I had the same issue yesterday and opened an new issue for it #1508.
